src/rard/templates/research/partials/concordance_edition_section.html

```diff
@@ -5,7 +5,7 @@
 
     <div class="d-flex flex-column justify-content-between edition-select">
     {% bootstrap_field edition_form.edition %}
-    <h2 class="alphabetum btn-link new-edition-btn">Or create a new Edition >></h2>
```

Comment:
do we not need it to be in alphabetum anymore?

Reply:
I think the convention across the rest of the site is for titles and form labels to use alphabetum, but additional instructions use the sans-serif font. In any case I found the alphabetum class was overriding the small class because it sets font size to be large. Happy to be overruled though.

Reply:
I'm happy for it to be sans-serif